### PR TITLE
cohort 4: [freedom] initial draft

### PR DESCRIPTION
**Pull request by jonleung**
This pull request talks about the the philosophy of Hack Camp.

I initially requested these changes to be made in #419.
I ran into some git issues.
So instead I redid my changes in this new branch on an updated version of master.
Then I closed #419.

There should be no more git issues.

### CODE REVIEW

File: cohort_4/playbook/ACTIVITIES.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 <!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-generate-toc again -->
 **Table of Contents**
-
```
Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md:3]: There should be an empty line here.

File: cohort_4/playbook/ACTIVITIES.md [hunk 2]
```diff
@@ -107,6 +113,145 @@
...
+  help their situation.
+
+### Examples
+
+1. Figure out what they want. The easiest way to do this is ask.
+
+> facilitator: So what do you want to get out of Hack Camp after it's over?
+> hacker: I want to be able to make an app. That's why I came here and you're
+> not teaching us how to make apps. Even worse, you're not even teaching us.
+> Your are just telling us to Google things.
```
Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md]: This renders all as one line.

File: cohort_4/playbook/ACTIVITIES.md [hunk 3]
```diff
@@ -107,6 +113,145 @@
...
+>facilitator: So it seems that you want to be able to build mobile apps after
+>you've leave hack camp correct?
+>hacker: Yes!
+>facilitator: Cool! So when I was learning how to build my web projects, I had
+>no idea how to at the time. The only way I could figure it out was to ask this
+>really smart friend every time. Unfortunately he wasn't always around. He
+>eventually just told me to Google to find the answers to my questions and after
+>a while, I was a lot less dependent on him after that and could find many of
+>the answers to my own problems. But I really needed to use Google for a long
+>time before I got the hang of it.
```
Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md]: This renders all as one line.

File: cohort_4/playbook/ACTIVITIES.md [hunk 4]
```diff
@@ -107,6 +113,145 @@
...
+
+3. Tell them how they can get what they want by doing the thing.
+
+>At Hack Camp, one of the things that is really important to us is to make sure
+>that hackers are prepared to continue learning new things even after hack camp.
+>That's why we're teaching you how to Google, to make sure you can continue
+>discovering things on your own so you can pick it up a lot faster than me.
```
Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md]: Can you put spaces after the `>`?

File: cohort_4/playbook/ACTIVITIES.md [hunk 5]
```diff
@@ -107,6 +113,145 @@
...
+>facilitator: No, you don't have to, but I want you to show us what you made!
+
+What is good about this:
+
+- At least it feels like there is a little bit more freedom because you said
+  "you don't have to"
+- However, there the clause "I want you to show us what you made!".
+  - anytime you see the phrase "I want you" in front of anything and the person
+    (you) in this case, will feel like they did it to satisfy the facilitator
+    over their own satisfaction.
```
Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md:225]: Can you capitalize this sentence?

File: cohort_4/playbook/ACTIVITIES.md [hunk 6]
```diff
@@ -107,6 +113,145 @@
...
+
+More optimal:
+
+>facilitator: No you don't have to demo. Everything is optional here at Hack
+>Camp. I know you told me earlier that you're trying to make friends with more
+>people. I think that by demoing your project, you will gain much need
+>experience.
```
Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md]: Can you put spaces after `>`?

So like:

```
> hey! this is a
> blockquote.
```

File: cohort_4/playbook/ACTIVITIES.md [hunk 7]
```diff
@@ -107,6 +113,145 @@
...
+
+#### Not going to energizers
+
+Sub-optimal:
+
+>facilitator: Come on Nick, you have to get off of Call of Duty, we have to go
+>to energizers now, the entire class is waiting for you.
+>hacker: But I want to finish my game first
+>facilitator: No, but you don't have time to play! You can play when you get
+>home.
```
Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md]: This'll all be rendered as one line.

**Comment by maxwofford** [comment]
I can cover the revisions @zachlatta brought up.

**Comment by maxwofford** [comment]
@zachlatta Just covered all the syntax changes you brought up. Thanks for that!

Would you please review and merge this PR?

**Pull request merged**